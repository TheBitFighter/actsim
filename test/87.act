defproc test()
{
   pint x[5];
   int i, j;

   x[0] = 2;
   x[1] = 4;
   x[2] = 6;
   x[3] = 8;
   x[4] = 10;

   chp {
       i := 0;
     *[ [([]k:5: i=k -> j := x[k])]; log ("got ", j); i := i + 1 ]
   }
}

test t;
