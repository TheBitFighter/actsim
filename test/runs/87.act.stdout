[                  20] <>  got 2
[                  40] <>  got 4
[                  60] <>  got 6
[                  80] <>  got 8
[                 100] <>  got 10
[                 110] <>  Warning: all guards false; no probes/shared vars
